warehouse/models/mart/payments/_payments.yml

```diff
@@ -167,9 +167,16 @@ models:
         description: |
           `source_record_id` of the Cal-ITP defined organization (from `dim_organizations`) associated with this payments activity.
           The mapping of organization records to payments entities is manually maintained in a seed file.
-      - name: end_of_month_date
+      - name: end_of_month_date_pacific
         description: |
-          The last day of the month of the aggregation_datetime.
+          The last day of the month of the aggregation_datetime in Pacific Time (involves conversion of aggregation_datetime to Pacific Time).
+          This column is primarily for use in BI tooling to support consistent dates across activity types.
+          Aggregation activity happens throughout the month.
+        tests:
+          - not_null
+      - name: end_of_month_date_utc
+        description: |
+          The last day of the month of the aggregation_datetime in UTC.
           This column is primarily for use in BI tooling to support consistent dates across activity types.
```

Comment:
Should this description line for `end_of_month_date_utc` be modified to reflect updated patterns of use?

Reply:
pushed update:

> This column preserves the end of month date value for the original UTC date as received by Littlepay, however, `end_of_month_date_pacific` is primarily used in BI tooling instead to support consistent dates across activity types.